### ISSUE TITLE
Laravel 5 support

### ISSUE THREAD

**Issue by JanMesaric** [escapeboy/jraty#2]
Is there going to be any laravel 5 support anytime soon? Or maybe some instructions?

**Comment by escapeboy** [comment]
Yes. Thank you for reminder! In next 1-2 I will find time to make a L5 version with some improvements. Any suggestions are welcome :)

**Comment by webmastervinay** [comment]
waiting for your L5 package...

**Comment by JanMesaric** [comment]
Would love the same simplicity as before :) one of a few plug'n'play packages :+1:
